runtime/src/main/java/org/corfudb/runtime/view/workflows/AddNode.java

```diff
@@ -46,7 +46,12 @@ public AddNode(@NonNull String endpointToAdd, @NonNull CorfuRuntime runtime,
     protected UUID sendRequest(@NonNull ManagementClient managementClient) throws TimeoutException {
         // Bootstrap a management server first.
         Layout layout = new Layout(runtime.getLayoutView().getLayout());
-        runtime.getManagementView().bootstrapManagementServer(nodeForWorkflow, layout).join();
+        boolean bootstrapped =
+                runtime.getManagementView().bootstrapManagementServer(nodeForWorkflow, layout);
```

Comment:
shouldnt timeoutException thrown from bootstrapManagementServer be caught here?

Reply:
Outdated